aerofiles/igc/reader.py

```diff
@@ -5,5 +5,144 @@ class Reader:
     see http://carrier.csi.cam.ac.uk/forsterlewis/soaring/igc_file_format/igc_format_2008.html
     """
 
-    def __init__(self, fp=None):
-        self.fp = fp
+    def __init__(self):
+        pass
+
+    def read(self, fp):
+        logger_id, fix_records, task, dgps_records, event_records, satellite_records, security_records, header,\
+            fix_record_extensions, k_record_extensions, k_records, comment_records = (None,) * 12
```

Comment:
what does `(None,) * 12` do? does is initialize all vars with `None` or with `(None,)`?